roles/openshift_control_plane/templates/master.yaml.v1.j2

```diff
@@ -81,12 +81,20 @@ kubernetesMasterConfig:
     feature-gates:
     - PersistentLocalVolumes=true
     - VolumeScheduling=true
+{% endif %}
+{% if openshift_master_use_ephemeralstorage | bool %}
+    feature-gates:
+    - LocalStorageCapacityIsolation=true
 {% endif %}
   controllerArguments: {{ openshift.master.controller_args | default(None) | lib_utils_to_padded_yaml( level=2 ) }}
 {% if openshift_master_use_persistentlocalvolumes | bool %}
     feature-gates:
     - PersistentLocalVolumes=true
     - VolumeScheduling=true
+{% endif %}
+{% if openshift_master_use_ephemeralstorage | bool %}
+    feature-gates:
```

Comment:
This is problematic if both openshift_master_use_persistentlocalvolumes and openshift_master_use_ephemeralstorage are true.  Then there are two feature-gates blocks.

This  a problem many places in this PR.

Reply:
Yes; lots more to do here.